doc/attr.xml

```diff
@@ -337,8 +337,9 @@ gap> Length(SmallGeneratingSet(S));
   <Returns>A minimal generating set for a semigroup.</Returns>
   <Description>
 
-    <B>Warning:</B> currently, no methods are installed to compute these
-    attributes.<P/>
```

Comment:
I'm confused, this documentation existed before, but there are no methods installed? Would it not make more sense to not document (or indeed declare) attributes where there are no methods installed? Maybe I misunderstood something here, but this seems like an oddity to me.

Reply:
And the example below seems to contradict the assertion that there are no methods installed for `MinimalInverseMonoidGeneratingSet`. The distinction between there being a method that "computes" a minimal generating set, and one that simply knows (like a monogenic inverse semigroup) a minimal generating set is probably too subtle. I suggest improving this warning to make it clearer what is going on.